benchmarks/dynamo/common.py

```diff
@@ -1419,16 +1418,40 @@ def load(cls, model, example_inputs, mode):
             # delete example_outputs and reset memory stats here
             del example_outputs
             if current_device == "cuda":
-                torch.cuda.reset_peak_memory_stats()
```

Comment:
I remember I put this before `empty_gpu_cache` because otherwise we get a large bump in AOTI's memory compression ratio. I wasn't sure if it is accurate or artificial. May need more digging here.

Please share your dashboard result link as a comment.

Reply:
@desertfire Ask and ye shall receive; I ran one overnight at https://hud.pytorch.org/benchmark/compilers?dashboard=torchinductor&startTime=Tue%2C%2006%20May%202025%2014%3A30%3A33%20GMT&stopTime=Tue%2C%2013%20May%202025%2014%3A30%3A33%20GMT&granularity=hour&mode=inference&dtype=bfloat16&deviceName=cuda%20(a100)&lBranch=benjaminglass1/fixup-aot-inductor-performance-benchmarks&lCommit=07ca95916772a2c2f3776fe2f0e41be2412591dd&rBranch=main&rCommit=7243c69421cd0b868f3fa3b552c17e9c8b3023a1

It doesn't appear to affect memory compression ratio at all.

Reply:
All right, I will happily take it.